packages/flame_rive/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# CHANGELOG
+
+## [next]
+
+## [1.0.0-releasecandidate.1]
+
```

Comment:
There shouldn't be any empty lines here if we are going to follow the format we have in Flame